starter-project-web-vue2/components/zerubabel/Blogs.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <v-container>
+        <v-container>
+            <AddBlog/>
+        </v-container>
+        <v-container class="
+        blog-list d-flex flex-column">
+            <v-card v-for="blog in blogs" :key="blog._id" class="blog m-4 ma-5 elevation-9 shaped d-flex flex-column">
+                <v-card-title class="card-title">
+                    <nuxt-link :to="'/zerubabel/' + blog._id">{{blog.title}}</nuxt-link>
```

Comment:
[template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for better readability. `to:="/zerubabel/${blog.title}`